src/App/Fossa/VSI/DynLinked/Internal/Binary.hs

```diff
@@ -89,12 +85,13 @@ lddParseDependency = Just <$> (LocalDependency <$> (linePrefix *> ident) <* symb
 -- > linux-vdso.so.1 => (0x00007ffc28d59000)
 --
 -- In other words, such libraries are in the format @{name}{literal =>}{memory address}@.
+-- Sometimes, the => is optional.
```

Comment:
Do we have more details on this being optional?  Does it have meaning, is it missing on some versions, etc?

Reply:
Added context